WardenApi/src/main/java/com/salesforce/dva/warden/dto/Policy.java

```diff
@@ -1,5 +1,165 @@
 package com.salesforce.dva.warden.dto;
 
-public class Policy {
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
 
+
+import com.salesforce.dva.warden.entity.SuspensionLevel;
```

Comment:
This import should be removed.  The class doesn't exist  in the API's project.  It will cause the build to fail.